SIT.Manager/Views/MainView.axaml

```diff
@@ -26,7 +26,7 @@
 			</ui:TeachingTip.IconSource>
 		</ui:TeachingTip>
 
-		<StackPanel Name="InfoBarStackPanel" Grid.Row="0" x:FieldModifier="Public">
+		<StackPanel Name="InfoBarStackPanel" Grid.Row="0" x:FieldModifier="public">
```

Comment:
To be honest I don't think it even needs  x:FieldModifier so could just remove it.